hooks/src/index.js

```diff
@@ -206,7 +206,9 @@ function scheduleFlushAfterPaint() {
 if (typeof window !== 'undefined') {
 	afterPaint = (component) => {
 		if (!component._afterPaintQueued && (component._afterPaintQueued = true) && afterPaintEffects.push(component) === 1) {
+			/* istanbul ignore next */
 			if (options.requestAnimationFrame) {
+				/* istanbul ignore next */
```

Comment:
the comment above the `if`-statement should be enough. Istanbul works on the AST-level

Reply:
Does not seem to be the case here